### ISSUE TITLE
Player mode

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/surge#6225]
We could provide an easy and fast to use entrance into the world of Surge by means of having a switch between Player/Edit modes. The player mode would basically contain all header controls, and crucially a button to switch back to Edit mode (same button would switch to Play mode while in Edit mode, of course). Then we could provide Scene Voicing parameters (Play Mode, Polyphony, Scene Octave, Scene Pitch, Glide, Pitch Bend Range), all 8 macros, patch information in a frame rather than a popup, and a scope.

In addition, @baconpaul's suggestion was to pad out the available area with direct access parameters. These would be parameters that the sound designer chooses that are taken to the front panel of the Player mode verbatim, one to one, or optionally scaled between sound designer specified min-max range. Design of that workflow is TBD still, but here's a mockup that I did in Figma that shows how could this Player mode potentially look:

![image](https://user-images.githubusercontent.com/2393720/171431169-b2ee4eb0-4118-4142-bfcf-9368b136a399.png)

**Comment by VincyZed** [comment]
+1, it's a cool idea and would be useful for many newcomers, as one of the things I hear the most from new users is that the interface is intimidating at first. Even though XT2 will partly improve on that regard, having this as an option would be a plus.

**Comment by MilesParker** [comment]
Very cool idea from peanut gallery. In general I think that these really are different personas; sound design and sound performance and as far as I've seen not very many synths address both cases well. Making them into separate modes makes a ton of sense. I do think that having the transition be natural -- ie not an entirely different expereince -- would be something to keep in mind.

**Comment by planist79** [comment]
good idea, a bit like the performance view in NI Guitar Rig.
I would appreciate the Waveform view in edit mode as well.

**Comment by mkruselj** [comment]
Yes of course it would be there, but it would have to be exchangeable with another part of the GUI (tabbed with another section). We're full, layout-wise.

**Comment by planist79** [comment]
hehe.. yeah, that's the obvious space for different tabs ;)
